mappings/net/minecraft/client/render/item/ItemRenderer.mapping

```diff
@@ -42,7 +42,7 @@ CLASS net/minecraft/class_918 net/minecraft/client/render/item/ItemRenderer
 		ARG 4 stack
 		ARG 5 light
 		ARG 6 overlay
-	METHOD method_23181 getArmorVertexConsumer (Lnet/minecraft/class_4597;Lnet/minecraft/class_1921;ZZ)Lnet/minecraft/class_4588;
+	METHOD method_23181 getItemGlintConsumer (Lnet/minecraft/class_4597;Lnet/minecraft/class_1921;ZZ)Lnet/minecraft/class_4588;
 		ARG 0 vertexConsumers
```

Comment:
Why is this param named `provider` in other methods?

Reply:
Ah, sorry. It wasn't mapped in the other methods and I didn't remember what y'all like to call it.

It's a `VertexConsumerProvider` which gets shortened to `provider` in my own projects.

Reply:
Changed.

Though someone should maybe watershed these names, since the method to get a `VertexConsumer` is called `getBuffer`, it might be sensible to call it something like `VertexBuffer` or `LayerBuffer`